metals/src/main/scala/scala/meta/internal/metals/ammonite/Ammonite.scala

```diff
@@ -149,7 +149,8 @@ final class Ammonite(
       .getOrElse(
         AmmVersions(
           ammoniteVersion = BuildInfo.ammoniteVersion,
-          scalaVersion = BuildInfo.scala212
+          // TODO: https://github.com/scalameta/metals/issues/1908
+          scalaVersion = "2.12.11"
```

Comment:
This is the default version for Ammonite

Reply:
:+1: